### ISSUE TITLE
logout not working

### ISSUE THREAD

**Issue by mavencode01** [sczyh30/vertx-blueprint-microservice#14]
I tried to following your example to implement a gateway for my service.

The logout seems not to be working when the user goes back to a protected URL.

How can I fix this?

**Comment by sczyh30** [comment]
Yes the logout is not working as the auth example here is only a sample. I'll fix this in the next version. You may refer to the documentation of Keycloak: [http://www.keycloak.org/documentation.html](http://www.keycloak.org/documentation.html).

**Comment by sczyh30** [comment]
Related to #5
